nncf/openvino/quantization/quantize_ifmodel.py

```diff
@@ -258,7 +261,9 @@ def get_if_cond_input_name(model: ov.Model, if_node: NNCFNode) -> str:
         """
         name_to_node_mapping = {op.get_friendly_name(): op for op in model.get_ops()}
         ov_node = name_to_node_mapping[if_node.node_name]
-        return ov_node.input_values()[0].any_name
+        if not ov_node.inputs()[0].get_tensor().get_names():
+            ov_node.inputs()[0].get_tensor().set_names(set(["if_0_input"]))
```

Comment:
Why do we need to add some additional hardcoded names?

OpenVINO allows to work with model inputs/outputs without names.

Reply:
This PR is a hot fix of ticket 122100. Current implementation fully relies on the fact that tensors have names. These names are used to map outputs to corresponding tensors. 
The ticket with updating the current approach to not using tensor names will be created.

Reply:
Ok.